cli/README.md

```diff
@@ -15,11 +15,12 @@ Running the tests requires [docker](https://docs.docker.com/get-docker/) to be i
 
 We recommend installing in a project specific virtual environment created using a environment
 management tool such as [conda](https://docs.conda.io/en/latest/) or
-[virtualenv](https://virtualenv.pypa.io/en/latest/).
+[virtualenv](https://virtualenv.pypa.io/en/latest/). See detaled insctructions [here](virtual-environment.md)
 
 Then install in editable mode by running
 
 ```bash
+conda activate pixlVE
```

Comment:
I wouldn't specify how they set up their environments, as we've given the choice above and its implied

Reply:
resolved here 5f00eee